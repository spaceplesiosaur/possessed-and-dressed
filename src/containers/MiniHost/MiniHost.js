import React from 'react';
import './MiniHost.scss';

export const MiniHost = ({picture, name}) => {
  return (
    <div className="entryPage-miniHost-hostAvatar">
      <img src={picture} className="host-avatar-picture" alt="host-screaming"></img>
      <p className="host-avatar-name">{name}</p>
    </div>
  )
}

export default MiniHost;
