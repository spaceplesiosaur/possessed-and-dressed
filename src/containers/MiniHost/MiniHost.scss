$dark-text-color: #282c34;
$bright-header-color: #FA1E88;
$ghosty-pale-color: #A2B6B5;

.entryPage-miniHost-hostAvatar {
  border-radius: 5px;
  border: 1px $dark-text-color solid;
  display: flex;
  flex-direction: column;
  margin: 2rem;
  text-align: center;
  box-shadow: 0 4.5px 9px 0 rgba(250, 30, 136, 0.25), 0 7px 25px 0 rgba(0, 0, 0, 0.2);
}

.host-avatar-picture {
  max-height: 320px;
  max-width: 200px;
}

.host-avatar-name {
  font-size: 3em;
  font-weight: bold;
  background-color: $ghosty-pale-color;
}
