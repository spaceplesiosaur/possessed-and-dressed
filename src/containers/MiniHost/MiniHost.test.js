import React from 'react';
import MiniHost from './MiniHost'
import { shallow } from 'enzyme'

describe('MiniHost', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MiniHost
        picture="picture url"
        name="Blucifer"
      />)
  })
  it('should render with the correct data', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
