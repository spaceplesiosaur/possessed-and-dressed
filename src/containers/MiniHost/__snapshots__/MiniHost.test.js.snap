// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MiniHost should render with the correct data 1`] = `
<div
  className="entryPage-miniHost-hostAvatar"
>
  <img
    alt="host-image-screaming"
    className="host-avatar-picture"
    src="picture url"
  />
  <p
    className="host-avatar-name"
  >
    Blucifer
  </p>
</div>
`;
